resources/apigateway-domainnames.go

```diff
@@ -43,10 +43,25 @@ func (l *APIGatewayDomainNameLister) List(ctx context.Context, o interface{}) ([
 
 		for i := range output.Items {
 			item := &output.Items[i]
+			
+			// Get tags for the domain
+			tagsOutput, err := svc.GetTags(ctx, &apigateway.GetTagsInput{
+				ResourceArn: aws.String("arn:aws:apigateway:" + opts.Config.Region + "::/domainnames/" + *item.DomainName),
```

Comment:
Please switch to using the `ptr` library and `fmt.Sprintf` instead of string concat.